### ISSUE TITLE
mono doesn't wanna work with Lantern on my LMDE system...

### ISSUE THREAD

**Issue by cheez26** [evancwright/Lantern#2]
So, long story short, I tried following the instructions on your site several times and no dice. This is occurring on my Linux Mint Debian Edition install and it feels weird to have to use Wine instead of mono to open "Lantern.exe" file.

This is what I get when I try every time I try mono:
`Cannot open assembly '/home/cheez26/Documents/Lantern/Lantern.exe': File does not contain a valid CIL image.`

Is there something I'm missing?

**Comment by evancwright** [comment]
I Don’t have anything running Debian, so I can’t reproduce your problem. Did you actually Mono? It worked at one point, unit I upgraded to Visual Studio 2022.
Evan


Sent from Yahoo Mail for iPhone


On Tuesday, July 23, 2024, 6:19 PM, Chelsea E. ***@***.***> wrote:




So, long story short, I tried following the instructions on your site several times and no dice. This is occurring on my Linux Mint Debian Edition install and it feels weird to have to use Wine instead of mono to open "Lantern.exe" file.

This is what I get when I try every time:
Cannot open assembly '/home/cheez26/Documents/Lantern/Lantern.exe': File does not contain a valid CIL image.

Is there something I'm missing?

**Comment by evancwright** [comment]
This might have something to do with the need for .net 6.0 support. I am in training today, but I’ll try to see if I get Lantern running under Mono when I get home. Or, at least figure out what is wrong. What target platform are you aiming for?


Sent from Yahoo Mail for iPhone


On Tuesday, July 23, 2024, 9:00 PM, Evan Wright ***@***.***> wrote:

I Don’t have anything running Debian, so I can’t reproduce your problem. Did you actually Mono? It worked at one point, unit I upgraded to Visual Studio 2022.
Evan


Sent from Yahoo Mail for iPhone


On Tuesday, July 23, 2024, 6:19 PM, Chelsea E. ***@***.***> wrote:




So, long story short, I tried following the instructions on your site several times and no dice. This is occurring on my Linux Mint Debian Edition install and it feels weird to have to use Wine instead of mono to open "Lantern.exe" file.

This is what I get when I try every time:
Cannot open assembly '/home/cheez26/Documents/Lantern/Lantern.exe': File does not contain a valid CIL image.

Is there something I'm missing?

**Comment by cheez26** [comment]
> This might have something to do with the need for .net 6.0 support. I am in training today, but I’ll try to see if I get Lantern running under Mono when I get home. Or, at least figure out what is wrong. What target platform are you aiming for?

Well, Linux obviously. I'm just trying to get the IDE open. And yes, I did try mono, but like I said, no dice. I even tried it without sudo at one point and same result.
